arch/portduino/portduino.ini

```diff
@@ -1,6 +1,6 @@
 ; The Portduino based sim environment on top of any host OS, all hardware will be simulated
```

Comment:
While we're at it, this comment is outdated. Not purely a sim environment any more. Would you do us the honor to update it as well? :-)